distribution/lib/Standard/Table/0.0.0-dev/src/Data/Aggregate_Column.enso

```diff
@@ -175,31 +174,19 @@ type Aggregate_Column
     column_name : Table->Text
     column_name table =
         if this.new_name.is_nothing.not then this.new_name else
-            get_name c = (this.resolve_column table c).name
+            get_name c prefix:Text="" = prefix + (this.resolve_column table c).name
```

Comment:
What is the purpose of this optional prefix if every call `get_name c x` is equivalent to the (possibly a bit more readable, as direct) `x + get_name c`?